calicoctl/commands/node/run.go

```diff
@@ -192,12 +192,17 @@ Description:
 
 	// Create a mapping of environment variables to values.
 	envs := map[string]string{
-		"NODENAME": name,
+		"NODENAME":                  name,
 		"CALICO_NETWORKING_BACKEND": backend,
 		"NO_DEFAULT_POOLS":          noPoolsString,
 		"CALICO_LIBNETWORK_ENABLED": fmt.Sprint(!disableDockerNw),
 	}
 
+	// Set CALICO_LIBNETWORK_IFPREFIX env variable to "cali" by default if Docker network is enabled.
+	if !disableDockerNw {
+		envs["CALICO_LIBNETWORK_IFPREFIX"] = "cali"
```

Comment:
No - this doesn't fix the issue #841 because I can't change the prefix if I want to.

e.g What if I want the prefix to be "eth" - how do I do that?